src/sconsx/qt4.py

```diff
@@ -251,10 +251,16 @@ def locateQt4Command(env, command, qtdir) :
     env['QTDIR']  = _detect(env)
     # TODO: 'Replace' should be 'SetDefault'
 #    env.SetDefault(
+    qtinclude = os.path.join('$QTDIR', 'include')
```

Comment:
It would be nice to explain why we search a dir with qt and qt/QtCore.
Later on these lines have to be understandable...